renovate.json5

```diff
@@ -0,0 +1,36 @@
+{
+  extends: [
+    "config:base",
+    ":disableRateLimiting",
```

Comment:
`config:base` set `prHourlyLimit: 2` which mean renovate create 2 PR in a hour.

[disableRateLimiting](https://docs.renovatebot.com/presets-default/#disableratelimiting) make no limtation. If too many PR are create and fail CI, `prConcurrentLimit: 5` might be introduced.

Reply:
I think setting `prConcurrentLimit: 5` is reasonable otherwise we may have too many PRs to check. Let's set this for now and tune it later if needed.

Reply:
yep, added https://github.com/runatlantis/atlantis/pull/2807/commits/fe8dad4040dc651269b26a24a6653ae1417e9c07